config/playground/eks/dev/outputs.tf

```diff
@@ -0,0 +1,59 @@
+#
+# Outputs
```

Comment:
could we elaborate more what does output means?

Reply:
Yes. Add more context for the output